cucumber/api/features/resource_visibility.feature

```diff
@@ -1,7 +1,109 @@
 @logged-in
 Feature: Rules which govern the visibility of resources to roles.
 
-  Scenario: Resources from a foreign account are not visible
+  Scenario: Resources from a foreign account are not visible without a permission
     Given I create a new resource in a foreign account
+    And I create a new user "alice"
+
+    When I login as "alice"
+    And I successfully GET "/resources"
+
+    Then the resource list should not include the newest resource
+
+  Scenario: Resources from a foreign account can be visible
+    Given I create a new resource in a foreign account
+    And I create a new user "alice"
+    And I permit user "alice" to "try" it
+
+    When I login as "alice"
+    And I successfully GET "/resources"
+    Then the resource list should include the newest resource
+
+    When I successfully GET "/resources/cucumber"
```

Comment:
I think at this point it's better to document this as `GET /resources?account=cucumber`